server/remote/msgraph/check_user_status.go

```diff
@@ -0,0 +1,55 @@
+package msgraph
+
+import "strings"
+
+func (c *client) CheckUserStatus() bool {
+	// Not checking for API calls made using super client
+	if c.store == nil {
+		return true
+	}
+
+	user, err := c.store.LoadUser(c.mattermostUserID)
```

Comment:
I still think this code belongs in a different package than `remote/msgraph`. This code doesn't have much to do with `msgraph`, compared to the business logic of the store etc. This package should not be aware of the store.

Reply:
See https://github.com/mattermost/mattermost-plugin-mscalendar/pull/256#discussion_r1190320663 on a solution for this